packages/babel-jest/src/index.ts

```diff
@@ -39,10 +38,13 @@ const createTransformer = (options: any): Transformer => {
     sourceMaps: 'both',
   };
 
+  // @ts-ignore: seems like this is removed. Is that true?
   delete options.cacheDirectory;
```

Comment:
looks like there's no such thing indeed: https://babeljs.io/docs/en/options

Reply:
seems to come from babel-loader: https://github.com/babel/babel-loader